OSDOCS#8714: OSUS 5.0.3 release image signature changes (#74995)
openshift/openshift-docs: updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

====

// Configuring the Cluster Version Operator (CVO)
include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
include::modules/update-service-configure-cvo.adoc[leveloffset=+1]

[NOTE]
====
See xref:../../../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[Configuring the cluster-wide proxy] to configure the CA to trust the update server.
====

// Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
include::modules/configure-cvo-retrieve-image-signatures.adoc[leveloffset=+2]

[id="next-steps_updating-restricted-network-cluster-osus"]
== Next steps

Before updating your cluster, confirm that the following conditions are met:

* The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.

> * The release image signature config map for the new release is applied to your cluster.

Reviewer: original signature config map is still support, so i think we should not remove it, right? so suggest to add `or the cluster is able to retrieve release image signatures for the update target release images from osus`

Author: I can keep the old statement while including the new option in the same sentence, thanks

[NOTE]
====

> Release image signatures allow the Cluster Version Operator (CVO) to ensure the integrity of release images by verifying that the actual image signatures match the expected signatures.

Reviewer 2: It's hard for me to understand this text :( Any one have any idea to make it easy to understand? cc @jiajliu @shellyyang1989

Author: I can rephrase this sentence if it's hard to understand. I think it's explaining how the CVO uses release image signatures to verify that the release images you are trying to apply during an update match the images that Red Hat serves (and thus haven't been modified by someone else).

Maybe the note could be rephrased to this:
```suggestion
The Cluster Version Operator (CVO) uses release image signatures to ensure that release images have not been modified, by verifying that the release image signatures match the expected result.
```

Let me know if that rephrasing would make more sense to you.

Reviewer 2: Cool! Right now it is very clear for me. Thank you @skopacz1

See xref:../../../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[Configuring the cluster-wide proxy] to configure the CA to trust the update server.
====

// Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
include::modules/configure-cvo-retrieve-image-signatures.adoc[leveloffset=+1]

[id="next-steps_updating-restricted-network-cluster-osus"]
== Next steps

Before updating your cluster, confirm that the following conditions are met:

* The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.

> * Your cluster is able to retrieve release image signatures for the update target release images.

Author: Rather than adding an additional statement about how oc mirror 4.15+ and OSUS 5.0.3+ makes this step unnecessary, I just broadened this condition to state that the cluster can get access to signatures somehow. 

If it's worth specifying the two different ways that can happen depending on your workflow, I can add that in level two bullets underneath this one.
